src/doc/artist/snippet-api.md

```diff
@@ -875,35 +864,76 @@ Corresponding controller:
       return (t >>> 0) / 4294967296
     }
   }
-  var fxrand = sfc32(...hashes)
+  let rndHash = (n) =>
+    Array(n)
+      .fill(0)
+      .map((_) => alphabet[(Math.random() * alphabet.length) | 0])
+      .join("")
+  let matcher = (str, start) =>
+    str
+      .slice(start)
+      .match(new RegExp(".{" + ((str.length - start) >> 2) + "}", "g"))
+      .map(b58dec)
+  // make fxrand from hash
+  var fxhash = search.get("fxhash") || "oo" + rndHash(49)
+  var fxrand = sfc32(...matcher(fxhash, 2))
   // make fxrandminter from minter address
-  var fxminter =
-    search.get("fxminter") ||
-    "tz1" +
-      Array(33)
-        .fill(0)
-        .map((_) => alphabet[(Math.random() * alphabet.length) | 0])
-        .join("")
-  let fxminterTrunc = fxminter.slice(3)
-  regex = new RegExp(".{" + ((fxminterTrunc.length / 4) | 0) + "}", "g")
-  hashes = fxminterTrunc.match(regex).map((h) => b58dec(h))
-  var fxrandminter = sfc32(...hashes)
-
+  var fxminter = search.get("fxminter") || "tz1" + rndHash(33)
+  var fxrandminter = sfc32(...matcher(fxminter, 3))
   // true if preview mode active, false otherwise
   // you can append preview=1 to the URL to simulate preview active
   var isFxpreview = search.get("preview") === "1"
   // call this method to trigger the preview
   function fxpreview() {
-    window.dispatchEvent(new Event("fxhash-preview"))
-    setTimeout(() => fxpreview(), 500)
+    console.log("FXPREVIEW")
+    // window.dispatchEvent(new Event("fxhash-preview"))
```

Comment:
should this be commented out or not hahaha 😅 @maerzhase

Reply:
No we leave it
It's low key for them to check when the capture occurs in the console, and understand that we are running a dedicated event.
If anything we may want to generalize this code to always send the event